File: android/EpubRnD/src/com/hurix/epubRnD/VOs/WebViewDAO.java
package com.hurix.epubRnD.VOs;

public class WebViewDAO 
{
	private ChapterVO _chapterVO;
	private int indexOfPage;
	private int indexOfChapter;
	private int pageCount;
	
	public ChapterVO getChapterVO() {
		return _chapterVO;
	}

	public void setChapterVO(ChapterVO _chapterVO) {
		this._chapterVO = _chapterVO;
	}

	public int getIndexOfPage() {
		return indexOfPage;
	}

	public void setIndexOfPage(int indexOfPage) {
		this.indexOfPage = indexOfPage;
	}

	public int getIndexOfChapter() {
		return indexOfChapter;
	}

	public void setIndexOfChapter(int indexOfChapter) {
		this.indexOfChapter = indexOfChapter;
	}

	public int getPageCount() {
		return pageCount;
	}

	public void setPageCount(int pageCount) {
		this.pageCount = pageCount;
	}

}

File: ios/EpubRnD/EpubRnD/WebViewDAO.m
//
//  WebViewDAO.m
//  EpubRnD
//
//  Created by UdaySravan K on 10/06/14.
//  Copyright (c) 2014 hurix. All rights reserved.
//

#import "WebViewDAO.h"

@implementation WebViewDAO

- (void) setIndexOfPage:(NSInteger) indexOfPage
{
    mIndexOfPage = indexOfPage;
}

- (NSInteger) getIndexOfPage
{
    return mIndexOfPage;
}

- (void) setIndexOfChapter:(NSInteger) indexOfChapter
{
    mIndexOfChapter = indexOfChapter;
}

- (NSInteger) getIndexOfChapter
{
    return mIndexOfChapter;
}

- (void) setPageCount:(NSInteger) pageCount
{
    mPageCount = pageCount;
}

- (NSInteger) getPageCount
{
    return mPageCount;
}

- (void) setFirstWordID:(NSInteger) firstWordIDofPage
{
    firstWordID = firstWordIDofPage;
}
- (NSInteger) getFirstWordID
{
    return firstWordID ;
}
- (void) setLastWordID:(NSInteger) lastWordIDofPage
{
    lastWordID = lastWordIDofPage;
}
- (NSInteger) getLastWordID
{
    return lastWordID;
}

- (void) setBookmarked:(BOOL) status
{
    isBookmarked = status;
}
- (BOOL) isBookmarked
{
    return isBookmarked;
}

@end
